src/content/docs/it/contribute.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Contribuisci ad Astro
+description: Come partecipare e contribuire ad Astro.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Siamo aperti a qualsiasi tipo di contributo che sia grande o piccolo. In quanto progetto open-source, crediamo nel ritorno ai nostri collaboratori. Siamo felici di aiutarvi con indicazioni sulle PR, sulla scrittura tecnica e sulla trasformazione di qualsiasi idea in realtà.  
+
+Vuoi essere ancora più coinvolto? Consulta il nostro [documento di governance](https://github.com/withastro/.github/blob/main/GOVERNANCE.md) per avere descrizioni dettagliate dei diversi ruoli, processi di nomina dei maintainer, processi di revisione del codice e applicazione del Codice di Condotta.
+
+## Modi per Contribuire
+
+### Repository del progetto
+
+Ci sono molti modi per contribuire al Progetto Astro! Ogni repository di Astro ha un README con un link a un file `CONTRIBUTING.md` nel codice del progetto.
+
+Visita il [profilo GitHub di Astro](https://github.com/withastro) per trovare i repository relativi a:
+
+- Il [codice principale di Astro](https://github.com/withastro/astro), inclusi le integrazioni ufficiali e i modelli di avvio.
```

Comment:
I'm not sure a reader will understand what a "modello di avvio" is, just by reading this sentence. I would either keep the english term "starter template" or maybe "template di avvio". In my opinion, the term "template" is widely used in the developers' community.
What do you think?

Reply:
Or "template di creazione"

Reply:
Actually, it doesn't sound very good. Maybe it would be better to use a different term. I could use 'template' as the word, but if anyone has other ideas, I would be happy to receive them.